CorsixTH/Src/th_sound.cpp

```diff
@@ -37,12 +37,30 @@ sound_archive::~sound_archive()
     delete[] data;
 }
 
+namespace {
+
+uint32_t bytes_to_uint32_le(const uint8_t* bytes)
```

Comment:
Could use a little documentation I guess, although it's hardly complicated at all.